modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaClientCodegen.java

```diff
@@ -84,6 +84,9 @@
     public static final String VERTX = "vertx";
     public static final String MICROPROFILE = "microprofile";
     public static final String APACHE = "apache-httpclient";
+    public static final String MICROPROFILE_REST_CLIENT_VERSION = "microprofileRestClientVersion";
+    public static final String MICROPROFILE_REST_CLIENT_DEFAULT_VERSION = "2.0";
```

Comment:
@wing328 it would be good, but I found only one provider that supports MP REST client 3.0 (Jersey).  If you think that it is OK I can make that change as the part of this PR.